main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name = "George", last_name = "Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
```

Comment:
The update operation for 'Kianu Reaves' is incorrect. Ensure you filter the correct actor before updating to 'Keanu Reeves'. This is crucial for the functionality of the code.